tiatoolbox/data/pretrained_model.yaml

```diff
@@ -611,6 +611,27 @@ fcn_resnet50_unet-bcss:
             stride_shape: [256, 256]
             save_resolution: {'units': 'mpp', 'resolution': 0.25}
 
+unet_tissue_mask_tsef:
```

Comment:
what is tsef?

Reply:
Tissue segmentation excluding fat as stated in https://arxiv.org/pdf/2202.09971.pdf.

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/ee13a1f6f90b392278ac14b6c28d7d8465c083b7/tiatoolbox/data/pretrained_model.yaml#L615-L620